sites/website/src/docs/resources/cheat-sheet.md

```diff
@@ -0,0 +1,695 @@
+---
+id: cheat-sheet
+title: Cheat Sheet
+sidebar_label: Cheat Sheet
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/resources/cheat-sheet.md
+---
+
+# Cheat Sheet
+
+## Packages:
+
+### [@microsoft/fast-element](https://www.fast.design/docs/fast-element/getting-started)
+
+**A library that is a lightweight means to easily build performant, memory-efficient, standards-compliant Web Components.**
+
+* Provides a thin layer of opinions on top of [Web Components](https://www.fast.design/docs/resources/why-web-components), lifting the level of abstraction just enough to make it easier and FASTer to [build components]((#building-components)).   
+* Use this library when you want to create new custom web components. 
+
+To install the fast-element library, use either `npm` or `yarn`:
+
+```shell
+npm install --save @microsoft/fast-element
+```
+
+```shell
+yarn add @microsoft/fast-element
+```
+
+Within your JavaScript or TypeScript code, you can then import library APIs like this:
+
+```ts
+import { FASTElement } from "@microsoft/fast-element";
+```
+
+---
+
+### [@microsoft/fast-foundation](https://www.fast.design/docs/introduction)
+
+**A library that provides foundational building blocks that can be assembled to create new design systems and component libraries.**
```

Comment:
Technically I think this still leaves a knowledge gap that I'd love to figure out how exactly to close. This is the "larger" and likely most common purpose, but Foundation also provides a set of common primitive classes and templates which represent functional elements. In the case of something like a "color swatch", Foundation gives us reusable radio/radiogroup bits which can then be styled to meet this scenario. I don't think there's anything wrong with just focusing on this portion, but Foundation enables so much and it seems like the quick hit here is scoped to two things (which area kind of one most of the time...).

Reply:
@chrisdholt Thank you - I updated with verbiage from here:

![Screen Shot 2021-12-15 at 4 32 13 PM](https://user-images.githubusercontent.com/35910788/146286242-7adcc5f5-5fd3-48ea-b9a8-dbbcc50da288.png)